hypervisor/include/arch/x86/trusty.h

```diff
@@ -116,7 +116,9 @@ struct key_info {
 
 struct secure_world_memory {
 	/* The secure world base address of GPA in SOS */
-	uint64_t base_gpa;
+	uint64_t base_gpa_map_sos;
```

Comment:
for these there fileds, do you mean these below?

base_gpa_in_sos (gpa from sos's pespective)
base_gpa_in_uos (gpa ... uos's ..)
base_hpa (hpa in vmm/host)

Reply:
Yes, alright. should I change the name?

Reply:
if my understanding is correct, why not keep naming consistent?